test-data/field-records.json

```diff
@@ -2,203 +2,221 @@
   "parent-address": {
     "entry-number": "20",
     "entry-timestamp": "2016-08-23T09:49:25Z",
-    "item-hash": "sha-256:30646573a1ee84e5ad39b4aaf0d31794228ad07117acb1c1584f36312ac94052",
-    "text": "An address containing an address",
-    "field": "parent-address",
-    "phase": "discovery",
-    "datatype": "string",
-    "register": "address",
-    "cardinality": "1"
```

Comment:
The actual field records json includes these properties (although they don't make any difference to the functionality of the serializer)

"index-entry-number": "20",
 "key": "parent-address",